components/automate-ui/src/app/entities/api-tokens/api-token.requests.ts

```diff
@@ -81,16 +81,21 @@ export class ApiTokenRequests {
     }
   }
 
-  public create(id: string, name: string, version: IAMMajorVersion = 'v1'):
+  public create(tokenData: CreateTokenPayload, version: IAMMajorVersion = 'v1'):
```

Comment:
[nit] if we destructure this here,
```suggestion
  public create({ id, name , projects }: CreateTokenPayload, version: IAMMajorVersion = 'v1'):
```
we can pass `{ id, name, projects, active: true }` below.